.github/workflows/unit-test.yml

```diff
@@ -83,8 +83,17 @@ jobs:
         chmod -R 767 ./ # TODO: Possibly integrate in wp-env
         npm run wp-env start
 
-    - name: Running the tests
-      run: npm run test-php && npm run test-unit-php-multisite
+    - name: Running lint check
+      run: npm run lint-php
+
+    - name: Running single site unit tests
+      run: npm run test-unit-php
+      if: ${{ success() || failure() }}
```

Comment:
could also use `always()` here, but this should basically mean "run this all the time unless the run was cancelled"